packages/teams-js/src/public/media.ts

```diff
@@ -204,14 +228,12 @@ export namespace media {
       ensureInitialized(runtime, FrameContexts.content, FrameContexts.task);
       if (!isCurrentSDKVersionAtLeast(mediaAPISupportVersion)) {
         const oldPlatformError: SdkError = { errorCode: ErrorCode.OLD_PLATFORM };
-        /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-        callback(oldPlatformError, null);
+        callback(oldPlatformError, new Blob());
```

Comment:
>new Blob() [](http://example.com/codeflow?start=35&length=10)

Similar to examples above, an empty `Blob` is used and now documented for the error case.